src/App.js

```diff
@@ -1,15 +1,35 @@
 import React from 'react';
+import {
+  BrowserRouter as Router, Route, Routes, Link,
+} from 'react-router-dom';
+import Home from './components/Home';
 import Calculator from './components/Calculator';
-import './App.css';
 import Quotes from './components/Quote';
+import './navbar.css';
 
-function App() {
-  return (
-    <div className="App">
-      <Quotes />
-      <Calculator />
-    </div>
-  );
-}
+const App = () => (
+  <Router>
+    <nav className="navbar">
+      <h1>Awesome Books</h1>
+      <ul className="navbar-links">
```

Comment:
[Optional] I suggest that you use a title that matches the caption of the project.

![image](https://user-images.githubusercontent.com/20554583/232105691-bea30137-8897-4155-8b6a-47c0c6411455.png)